keyboards/treasure/type9s2/keymaps/default/rules.mk

```diff
@@ -0,0 +1 @@
+VIA_ENABLE = yes
```

Comment:
Please delete this file. VIA should not be enabled on the default keymap.

Reply:
Thanks, The file has been deleted